riva/clients/asr/riva_asr_client_helper.cc

```diff
@@ -16,7 +27,10 @@ ReadPhrasesFromFile(const std::string& phrases_file)
     if (infile.is_open()) {
       std::string phrase;
       while (getline(infile, phrase)) {
-        phrases.push_back(phrase);
+        std::string trimmed_phrase = Trim(phrase);
```

Comment:
This can be done in place using single line regex replace function. Will be preferable to use this.

`line = std::regex_replace(line, std::regex("^ +| +$"), "");`